Cannot find module '../package.json'

Hello, am I having an issue running an app when  gatsby-source-cloudinary ins installed:

```
 Error: Cannot find module '../package.json'
  Require stack:
  - /home/chris/development/app/app-static/node_modules/gatsby-source-cloudinary/utils.js
  - /home/chris/development/app/app-static/node_modules/gatsby-source-cloudinary/gatsby-node.js
  
  - loader:1075 Function.Module._resolveFilename
    node:internal/modules/cjs/loader:1075:15
  
  - loader:920 Function.Module._load
    node:internal/modules/cjs/loader:920:27
  
  - loader:1141 Module.require
    node:internal/modules/cjs/loader:1141:19
  
  - helpers:110 require
    node:internal/modules/cjs/helpers:110:18
  
  - utils.js:4 Object.<anonymous>
    [app-static]/[gatsby-source-cloudinary]/utils.js:4:19
  
  - loader:1254 Module._compile
    node:internal/modules/cjs/loader:1254:14
  
  - loader:1308 Object.Module._extensions..js
    node:internal/modules/cjs/loader:1308:10
  
  - loader:1117 Module.load
    node:internal/modules/cjs/loader:1117:32
  
  - loader:958 Function.Module._load
    node:internal/modules/cjs/loader:958:12
  
  - loader:1141 Module.require
    node:internal/modules/cjs/loader:1141:19
  
  - helpers:110 require
    node:internal/modules/cjs/helpers:110:18
  
  - gatsby-node.js:5 Object.<anonymous>
    [app-static]/[gatsby-source-cloudinary]/gatsby-node.js:5:5
  
  - loader:1254 Module._compile
    node:internal/modules/cjs/loader:1254:14
  
  - loader:1308 Object.Module._extensions..js
    node:internal/modules/cjs/loader:1308:10
  
  - loader:1117 Module.load
    node:internal/modules/cjs/loader:1117:32
  
  - loader:958 Function.Module._load
    node:internal/modules/cjs/loader:958:12
  

not finished load plugins - 0.644s

error Command failed with exit code 1.
```

Config is:

```
{
      resolve: `gatsby-source-cloudinary`,
      options: {
        cloudName: process.env.CLOUDINARY_CLOUD_NAME,
        apiKey: process.env.CLOUDINARY_API_KEY,
        apiSecret: process.env.CLOUDINARY_API_SECRET,
        resourceType: `image`,
        // type: `twitter`,
        // maxResults: 22,
        // tags: true,
        // context: true,
        // prefix: `demo/animals`
      },
    },
```

ENV variables are defined

--- a/plugin/utils.js
+++ b/plugin/utils.js
@@ -1,7 +1,7 @@
 const cloudinary = require('cloudinary').v2;
 const { snakeCase } = require('lodash');
 
-const pluginPkg = require('../package.json');
+const pluginPkg = require('./package.json');
 const gatsbyPkg = require('gatsby/package.json');
 
 const SDK_CODE = 'X';

--- a/plugin/utils.test.js
+++ b/plugin/utils.test.js
@@ -1,4 +1,4 @@
-jest.mock('../package.json', () => ({
+jest.mock('./package.json', () => ({
   version: '0.1.6',
 }));
 jest.mock('gatsby/package.json', () => ({

